src/dodal/devices/enum_device.py

```diff
@@ -10,7 +11,7 @@
 StrictEnumT = TypeVar("StrictEnumT", bound=EnumTypes)
 
 
-class EnumDevice(StandardReadable, Generic[StrictEnumT]):
+class EnumDevice(StandardReadable, Movable, Generic[StrictEnumT]):
```

Comment:
nit: you can do this now. 
```suggestion
class EnumDevice(StandardReadable, Movable[StrictEnumT]):
```

Reply:
Good spot. Done!